packages/core/src/command/consolidated/consolidated-create.ts

```diff
@@ -35,20 +35,72 @@ export async function createConsolidatedFromConversions({
   patientId,
   destinationBucketName = getConsolidatedLocation(),
   sourceBucketName = getConsolidatedSourceLocation(),
-  logMemUsage,
 }: ConsolidatePatientDataCommand): Promise<Bundle> {
   const { log } = out(`createConsolidatedFromConversions - cx ${cxId}, pat ${patientId}`);
 
-  const consolidated = buildConsolidatedBundle();
+  const [conversions, docRefs] = await Promise.all([
+    getConversions({ cxId, patientId, sourceBucketName }),
+    getDocumentReferences({ cxId, patientId }),
+  ]);
+  log(`Got ${conversions} resources from conversions`);
 
-  const inputBundles = await listConversionBundles({ cxId, patientId, sourceBucketName, log });
-  if (!inputBundles || inputBundles.length < 1) {
+  const withDups = buildConsolidatedBundle();
+  withDups.entry = [...conversions, ...docRefs.map(buildBundleEntry)];
+  withDups.total = withDups.entry.length;
+  log(`Added ${docRefs.length} docRefs, to a total of ${withDups.entry.length} entries`);
+
+  log(`Deduplicating consolidated bundle...`);
+  const deduped = deduplicate({ cxId, patientId, bundle: withDups });
+  log(`...done, from ${withDups.entry?.length} to ${deduped.entry?.length} resources`);
+
+  const dedupDestFileName = createConsolidatedDataFilePath(cxId, patientId, true);
+  const withDupsDestFileName = createConsolidatedDataFilePath(cxId, patientId, false);
+  log(`Storing consolidated bundle on ${destinationBucketName}, key ${dedupDestFileName}`);
+  log(`Storing consolidated bundle w/ dups on ${destinationBucketName}, key ${dedupDestFileName}`);
+  await Promise.all([
```

Comment:
why are we awaiting these? couldn't we do them async...?

Reply:
1. This runs on a Lambda, so everything needs to be sync
2. We need those on S3 further downstream